### ISSUE TITLE
Oriental Outfits Quest.

### ISSUE THREAD

**Issue by Kertie** [kito2/mtibia-bounties#1137]
I started Oriental Outfits Quest and suddenly, Marina doesn't say anything about Calassa. (We should fix it)

![image](https://user-images.githubusercontent.com/45070560/59162863-96522880-8af8-11e9-864b-7ca493ca2473.png)

![image](https://user-images.githubusercontent.com/45070560/59162865-a0742700-8af8-11e9-8242-1b89c177b0e6.png)

However, you can travel to Calassa. So, let's continue the quest.

![image](https://user-images.githubusercontent.com/45070560/59163021-6193a080-8afb-11e9-9a27-e97e3f213198.png)

**Comment by Kertie** [comment]
1. I went to talk to Habdel in Darashia. He told me to visit Marina in Meriana.

2. I went to talk to Marina in Meriana. She told me that I must finish the Friends and Traders Quest with her.
![image](https://user-images.githubusercontent.com/45070560/64300164-182ce300-cf7c-11e9-900d-18ba9a4024e9.png)

3. I finished The Mermaid Marina Quest (her section on the Friends and Traders Quest - which gives you access to deal with Marina and exchange spider silks into spool of yarn)
![image](https://user-images.githubusercontent.com/45070560/64300309-9e492980-cf7c-11e9-8d12-7dfd7a41fae1.png)

4. I tried to continue the quest. She still told me that I must finish the Friends and Traders Quest.
![image](https://user-images.githubusercontent.com/45070560/64300319-a30ddd80-cf7c-11e9-8d30-2bd8956b9ee0.png)
